storage/innobase/xtrabackup/doc/source/advanced/log_enhancements.rst

```diff
@@ -0,0 +1,70 @@
+.. _logs:
+
+================================================================================
+Improved Log statements
+================================================================================
+
+**Percona XtraBackup** is an open-source command-line utility. Command-line tools have limited interaction with the background operations and the logs provide the progress of an operation or more information about errors.
+
+In earlier versions, the log statement did not have a standard structure. Notice in the following examples the variance in the log statements:
+
+* The backup log statement header has the name of the module, ``xtrabackup``, which generated the statement but no timestamp: 
+
+  .. code-block:: mysql
+    
+     xtrabackup: recognized client arguments: --parallel=4 --target-dir=/home/etc/mybackup/pxb-8/bld/backup --backup=1 --user=root --socket=/home/etc/mybackup/PS-8.0-trunk/bld/mysql-test/var/tmp/mysqld.1.sock --xtrabackup-plugin-dir=/home/etc/mybackup/pxb-8/bld/plugin_output_directory 
+
+    ./runtime_output_directory/xtrabackup version 8.0.27-19 based on MySQL server 8.0.27 Linux (x86_64) (revision id: b0f75188ca3)
```

Comment:
instead of runtime_output_directory use ./bin/

Reply:
Updated to bin